accounts/tasks.py

```diff
@@ -38,7 +38,7 @@ def update_portfolio_value():
         portfolio_value = user.current_portfolio_value()
         if user.portfolio_value != portfolio_value:
             user.portfolio_value = portfolio_value
-            user.save(update_fields=['portfolio_value'])
+            user.save()
```

Comment:
Are you sure is it safe? You want to save all user values asynchronously in task?